tailormap-components/projects/core/src/lib/workflow/workflows/Workflow.helpers.ts

```diff
@@ -0,0 +1,48 @@
+import { GeoJSONGeometry } from 'wellknown';
```

Comment:
Nitpicking: the current convention is to use lowercase names for files (at least start lower case) + helper instead of helpers

Reply:
You mean the file "Workflow.helpers.ts" itself (line 1 is selected here, which has a lowercase as import)?
Didn't know this convention, but I'll use it